viewer/src/datamodels/cad/rendering/EffectRenderManager.ts

```diff
@@ -333,8 +354,8 @@ export class EffectRenderManager {
     renderer.render(this._compositionScene, this._orthographicCamera);
   }
 
-  private renderAntiAliasToCanvas(renderer: THREE.WebGLRenderer) {
-    renderer.setRenderTarget(null);
+  private renderAntiAliasToTarget(renderer: THREE.WebGLRenderer, target: THREE.WebGLRenderTarget | null) {
+    renderer.setRenderTarget(target ?? null);
```

Comment:
```suggestion
    renderer.setRenderTarget(target);
```

target's type is already Target or null, why to do it here?